__tests__/seed.test.js

```diff
@@ -44,5 +44,41 @@ describe("GET /api/topics", () => {
           expect(typeof topic.description).toBe("string");
         });
       });
-  })
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("GET200: Endpoint responds with selected article by its ID", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(typeof article.author).toBe("string");
+        expect(typeof article.title).toBe("string");
+        expect(typeof article.article_id).toBe("number");
+        expect(typeof article.body).toBe("string");
+        expect(typeof article.topic).toBe("string");
+        expect(typeof article.created_at).toBe("string");
+        expect(typeof article.votes).toBe("number");
+        expect(typeof article.article_img_url).toBe("string");
+      });
+  });
+  test("GET400: Invalid ID Input", () => {
+    return request(app)
+      .get("/api/articles/99")
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Invalid ID Input");
```

Comment:
this should be a 404, and I would change the error message slightly because it's not quite invalid, it's valid but just non-existent